tests/integration/facets.ts

```diff
@@ -7,7 +7,7 @@ it('can select a facet option', () => {
   const [initialState, facetOption] = createInitialState(false);
   const answers = createMockedHeadless({}, initialState);
   answers.setFacetOption('testFieldId', facetOption, true);
-  expect(answers.state.filters.facets[0].options[0]).toEqual({
+  expect(answers.state.filters.facets?.[0].options[0]).toEqual({
```

Comment:
Out of curiosity did these TS issues come from a previous commit?

Reply:
yup, I can see these TS issues when I switch to develop branch just now.